src/services/completions.ts

```diff
@@ -5836,8 +5839,28 @@ function isProbablyGlobalType(type: Type, sourceFile: SourceFile, checker: TypeC
     return false;
 }
 
-function isStaticProperty(symbol: Symbol) {
-    return !!(symbol.valueDeclaration && getEffectiveModifierFlags(symbol.valueDeclaration) & ModifierFlags.Static && isClassLike(symbol.valueDeclaration.parent));
+function isStaticProperty(symbol: Symbol) {
+    return !!(symbol.valueDeclaration && getEffectiveModifierFlags(symbol.valueDeclaration) & ModifierFlags.Static && isClassLike(symbol.valueDeclaration.parent));
+}
+
+function isNativeFunctionMethod(symbol: Symbol, typeChecker: TypeChecker, type: Type): boolean {
```

Comment:
The only thing you need here is the symbol. Check that its *binding* parent is an interface declaration in the global scope whose name is `Function` or `CallbableFunction`. That's it.